contents/docs/user-guides/support.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Support
+sidebar: Docs
+showTitle: true
+---
+
+PostHog provides enhanced support to paying customers. Our goal with this is:
+
+* Reduce time and effort of deploying and managing PostHog ongoingly
+* Reduce the risk of managing PostHog
+* Productize frequent
+
+Broadly, our Enterprise support is bespoke, whereas Open Source, Cloud and Scale are much more fixed.
+
+|Support type|Open Source|Cloud|Scale|Enterprise|
+|---|---|---|---|---|
+|[Community](#community-support)|✔️|✔️|✔️|✔️|
+|[Dedicated Slack](#dedicated-slack)|x|Minimum $2k/month spend|✔️|✔️|
+|[Dedicated email](#dedicated-email)|x|x|x|✔️|
+|[Training sessions](#training-sessions)|x|x|x|✔️|
+|[Deployment developer pairing](#deployment-developer-pairing)|x|x|✔️|✔️|
+|[Configuration assistance](#configuration-assistance)|x|x|x|✔️|
+|[Standard Terms and Conditions](#standard-terms-and-conditions)|MIT Licence|✔️|✔️|✔️|
+|[Master Services Agreement](#master-services-agreement-msa)|x|x|x|✔️|
```

Comment:
It's clearer to me as it is bespoke legal agreement sounds like it could be a contract for buying a house, writing a will or anything, whereas an MSA infers it is a commercial relationship for selling something to someone, and I think from the description we make the fact it can go through legal review clear.